docs/_ingest/kafka.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Ingest Members With Kafka
+nav_order: 0
+---
+
+# Ingest Members With Kafka
+
+Ingesting members into an Event Stream without too much overhead? 
+That's now possible via ingestion over Apache Kafka.
+
+## Getting Started
+
+To get started with ingesting members via Kafka, you need to have the following:
+* Kafka consumer configuration (in the Application Properties)
+* Event Stream configuration pointing to a Kafka topic (in the Admin API) 
+
+### Application Properties
+
+The Kafka consumer configuration can be set in the `application.properties` file.
+
+The most basic properties that are needed are:
+````yaml
+spring.kafka.consumer.bootstrap-servers=localhost:9092
+spring.kafka.consumer.group-id=my-group
```

Comment:
It's a little inconsistent with the rest of our approach to use `application.properties` instead of `application.yml`